data/json/vehicle_parts.json

```diff
@@ -3467,6 +3467,27 @@
         ]
     },{
         "type" : "vehicle_part",
+        "id" : "pivot",
+        "name" : "all-wheel steering",
```

Comment:
This is really just there to provide a way to say "I want the vehicle to rotate around HERE" for cases where center-of-mass isn't what you want - and having all-wheel steering is just the rationalization. If we can calculate it properly based on wheel positions etc, that's better, but I'm not sure how to do that.